tests/end2end/features/tabs.feature

```diff
@@ -255,6 +255,18 @@ Feature: Tab management
             - data/numbers/2.txt (active)
             - data/numbers/3.txt
 
+    Scenario: :tab-focus with 0
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new tab
+        And I open data/numbers/3.txt in a new tab
+        And I run :tab-focus 1
+        And I run :tab-focus 3
+        And I run :tab-focus 0
```

Comment:
This tests the index *argument* to `:tab-focus`, which is different from the count (which gets passed from keybindings, e.g. `0T`).

You can use `And I run :tab-focus with count 0` to test the count.